Q2ShawnCampbell.md

```diff
@@ -0,0 +1,148 @@
+# Quiz #2
+
+## Instructions
+
+1. Fork this repo
+2. Clone your fork
+3. Fill in your answers by writing in the appropriate area, or placing an 'x' in
+the square brackets (for multiple-choice questions).
+4. Add/Commit/Push your changes to Github.
+5. Open a pull request.
+
+## CSS
+
+### Question #1
+
+Describe the purpose of a clearfix in CSS, and give an example of how to do it.
+
+Your Answer:
+```The clearfix is a CSS method used to make it easier to format a group of
+images.
+```
+
+### Question #2
+
+What does the following selector do?  `ul.dropdown > li`?
+
+Select 1:
+```
+[] Selects all li's which are directly inside a ul of class dropdown (children)
+[X] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[] Selects all ul's of class dropdown, as well as the children elements that are li's
+[] Selects all ul's of class dropdown, only if their children are exclusively li's
+```
+
+## Scope/Context/Closures
+
+### Question #3
+
+Describe the rules of scope in JavaScript.
+
+Your Answer:
+```In Javascript, scope refers to the limits of using types of variables. Global
+variables, those not defined only by one function, can be used in any function.
+Local variables can only be used in specific functions that they were
+defined for, and created within.
+```
+
```

Comment:
Technically correct, although local variables can also be accessed by functions of higher scope. So the following code will run as intended...

```js
function outerOne( ){
  var thing = "doodad"
  function innerOne( ){
    console.log( thing );
  }
}
```